static/js/pages/certificates/CertificateGenerate.tsx

```diff
@@ -12,6 +15,15 @@ interface Certs {
 const CertificateGenerate: FC = () => {
   const [isGenerating, setGenerating] = useState(false);
   const [certs, setCerts] = useState<Certs | null>(null);
+  const { isAuthenticated, isAuthLoading } = useAuth();
```

Comment:
Not sure if the process here works. Would something trigger the new check if it is authenticated after following the steps? Maybe we need a button to verify it and notify in case of success instead of redirecting?

Reply:
It works, because I'm invalidating the query. I tested it many, many times. :)

Reply:
+1

Reply:
Sorry, I had a false memory about query invalidation. So I'll add further explanation of my understanding of what happens here: the query gets re-fetched on window focus. So everything is automatic: once you switch back to the browser window after running the `lxc config trust add` command in the terminal, you'll get a fresh fetch that will give you a 200 instead of 403, hence updating the UI "automatically".